### PR TITLE
use admin client instead of fetch

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: js/libs/keycloak-admin-client/src/resources/identityProviders.ts [hunk 1]
```diff
@@ -116,10 +116,11 @@ export class IdentityProviders extends Resource<{ realm?: string }> {
...
   public importFromUrl = this.makeRequest<
-    {
-      fromUrl: string;
-      providerId: string;
-    },
+    | {
+        fromUrl: string;
+        providerId: string;
+      }
+    | FormData,
```
Comment (jonkoops) [inline js/libs/keycloak-admin-client/src/resources/identityProviders.ts:123]: The `FormData` type can be anything. Are we doing this in other places as well? Seems kinda counterintuitive to go from a well typed API to a generic map of data.

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
@edewit does this close an issue? Can we link it?

File: js/libs/keycloak-admin-client/src/resources/identityProviders.ts (hunk 1 above)
Comment (edewit) [reply js/libs/keycloak-admin-client/src/resources/identityProviders.ts:123]: right, it's what the api expects a post with form data

**Comment by edewit** [comment]
I just moved the stuff that was being done in the admin-ui to the admin-client, there is no issue just a small refactor

**Pull request merged**